src/applications/caregivers/tests/components/FormReview/AddressWithAutofillReviewField.unit.spec.js

```diff
@@ -5,18 +5,6 @@ import { expect } from 'chai';
 import { AddressWithAutofillReviewField } from '../../../components/FormReview/AddressWithAutofillReviewField';
 
 describe('CG <AddressWithAutofillReviewField>', () => {
-  const defaultProps = {
-    canAutofillAddress: false,
-    formData: {
-      street: '1350 I St. NW',
-      street2: 'Suite 550',
-      city: 'Washington',
-      state: 'DC',
-      postalCode: '20005',
-      'view:autofill': false,
-    },
-    inputLabel: 'Primary Family Caregiver\u2019s',
-  };
```

Comment:
🤔 Is shifting this below `getSelectors` for some visual locality or something? AFAICT the content's the same, but I don't get the reordering....

Reply:
Yeah I just wanted it right above the subject so it was clearest where the defaultProps were being set.